endpoints/github.ts

```diff
@@ -111,10 +119,16 @@ const statesColor = {
 async function status ({ owner, repo, ref = 'master', context }: Args) {
   const resp = await restGithub(`repos/${owner}/${repo}/commits/${ref}/status`)
 
-  const state = typeof context === 'string'
-    ? resp!.statuses.find(st => st.context === context).state
+  let state = typeof context === 'string'
+    ? resp!.statuses.filter(st => st.context.startsWith(context))
     : resp!.state
 
+  if (Array.isArray(state)) {
+    const failure = state.find(x => x.state === 'failure')
```

Comment:
`state` for context could be `error`
https://developer.github.com/v3/repos/statuses/#get-the-combined-status-for-a-specific-ref

Reply:
Oookay, good catch.

Reply:
Actually, I'm not sure.

In the `statuses` endpoint Appveyor shows `error`, but it's not included int commit status endpoint, where in turn it is `success` :scream: 

Strange?

Check these:

![2019-06-22-173710_1280x1024_scrot](https://user-images.githubusercontent.com/5038030/59965281-90a30c80-9514-11e9-8e5a-57b56400e688.png)
![2019-06-22-173658_1280x1024_scrot](https://user-images.githubusercontent.com/5038030/59965282-90a30c80-9514-11e9-9e8a-03b454ebf5fe.png)

Reply:
https://api.github.com/repos/zeit/hyper/status/efeedd0a

The AppVeyor state is `success`

```
    {
      "url": "https://api.github.com/repos/zeit/hyper/statuses/efeedd0a9d3fe8126df9402bb326206f0a792147",
      "avatar_url": "https://avatars2.githubusercontent.com/oa/39145?v=4",
      "id": 6770055216,
      "node_id": "MDEzOlN0YXR1c0NvbnRleHQ2NzcwMDU1MjE2",
      "state": "success",
      "description": "AppVeyor build succeeded",
      "target_url": "https://ci.appveyor.com/project/zeit/hyper/builds/24432313",
      "context": "continuous-integration/appveyor/branch",
      "created_at": "2019-05-09T19:00:19Z",
      "updated_at": "2019-05-09T19:00:19Z"
    }
```